sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (Unreleased)
```

Comment:
I've heard it recommended you don't remove the preview version until you're ready to release, in case it accidentally leaks out (burns a version number).

Reply:
I have copied to 4.1.0 only the changes that have happened after the last non-preview version. I have also restored the existing information, but I realized that I forgot to add to master the changelog entries for the last non-preview versions, so I have added those.

Reply:
> I've heard it recommended you don't remove the preview version until you're ready to release, in case it accidentally leaks out (burns a version number).

This is us getting ready to release though :)
Making all the changes so that we can release on Monday/Tuesday

Reply:
Ok I removed the unreleased preview sections! Thank you 🌞